src/modules/ProfileLocation.rb

```diff
@@ -153,7 +153,7 @@ def Process
           raise Break if AutoinstConfig.scheme == "device"
         end
         if AutoinstConfig.scheme == "label"
-          Popup.Error(_("label not found while looking for autoyast profile"))
+          Report.Error(_("label not found while looking for autoyast profile"))
```

Comment:
Replace `Yast.import "Popup"` by `Yast.import "Report"` in the import section at the beginning.